packages/grafana-ui/src/components/Slider/RangeSlider.tsx

```diff
@@ -47,6 +47,7 @@ export const RangeSlider: FunctionComponent<RangeSliderProps> = ({
         onAfterChange={onAfterChange}
         vertical={!isHorizontal}
         reverse={reverse}
+        tabIndex={[0, 1]}
```

Comment:
what does this solve?

AFAIK this sets the tabIndex of the second handler to 1, which means users, after focusing the first handler, would have to tab all the way through the end of the page (or all the way through elements that have a tabindex != 0) to focus the second one.

if the slider ha some weird behavior, i'd fix it separtely in https://github.com/grafana/grafana/issues/53776 and focus on the tab component on this PR

Reply:
I am aware that this can only be a temporary solution but for now it's making the rest of the tab content accessible. Otherwise the search bar as well as the list of queries would continue not to be accessible by keyboard until I found a solution for the slider.

Reply:
then do you mind adding a  TODO/FIXME comment in the code specifying it's a temporary workaround and why?

Reply:
No, I don't mind. Done.